viaMover/web-app
# [Pages] Add more page (complete)

What was done
* Added new page
* Reverted the deletion of sections & cards
* Refactored views, legacy components
* Ported styles from `static_layouts` to the `_more.less`
* Added new component constructing `<picture>` HTML5 tag with all the valid data
  * Seems like it will be the most useful with `v-once` directive used, anyways saves so much time, ensures that all resources that are requested use `require(<resource>)` routine

How to test
* Enable these flags if disabled:
  * `isGovernanceEnabled`
  * `isMoreEnabled`
  * `isNibbleShopEnabled`
  * `isNftDropsEnabled`
* Check that appearance follows Figma
* Disable all the flags from previous steps, enable just `isMoreEnabled` and `isNftDropsEnabled`. Should be also displayed properly
* Disable `isMoreEnabled`, the dashboard page should lose `More` nav button, `/more` route should be disabled and leading to the 404

Notes/TODO
* The sections are filled with kind of dummy data. TBA: Vuex store-located entries (not in the scope of this PR)
